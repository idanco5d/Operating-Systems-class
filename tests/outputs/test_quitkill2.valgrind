==2344== Memcheck, a memory error detector
==2344== Copyright (C) 2002-2022, and GNU GPL'd, by Julian Seward et al.
==2344== Using Valgrind-3.20.0 and LibVEX; rerun with -h for copyright info
==2344== Command: /mnt/c/Users/illus/os/hw/1/smash
==2344== Parent PID: 2338
==2344== 
==2344== 
==2344== FILE DESCRIPTORS: 4 open (3 std) at exit.
==2344== Open file descriptor 3: /mnt/c/Users/illus/os/hw/1/tests/outputs/test_quitkill2.valgrind
==2344==    <inherited from parent>
==2344== 
==2344== 
==2344== HEAP SUMMARY:
==2344==     in use at exit: 0 bytes in 0 blocks
==2344==   total heap usage: 33 allocs, 33 frees, 80,072 bytes allocated
==2344== 
==2344== All heap blocks were freed -- no leaks are possible
==2344== 
==2344== For lists of detected and suppressed errors, rerun with: -s
==2344== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
