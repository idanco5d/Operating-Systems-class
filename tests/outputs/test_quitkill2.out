smash> smash> smash> smash: sending SIGKILL signal to 2 jobs:
9590: sleep 10000&
9591: sleep 10000&
